libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts

```diff
@@ -0,0 +1,121 @@
+    brandColor: '#8C6CFF',
+    description: 'Chunk text to a fixed length.',
+    summary: 'Chunk text',
+
+    inputs: {
+      type: 'object',
+      properties: {
+        chunkingStrategy: {
+          title: 'Chunking strategy',
+          type: 'string',
+          description: 'Chunking strategy to be used.',
+          default: 'TokenSize',
+          enum: ['TokenSize'],
+        },
+        text: {
+          title: 'Text',
+          description: 'Text to be chunked.',
+          type: 'string',
+        },
+        encodingModel: {
+          title: 'Encoding model',
+          type: 'string',
+          description: 'The encoding model to use fro chunking.',
+          default: 'cl100k_base',
+          'x-ms-editor': 'dropdown',
+          'x-ms-editor-options': {
+            multiSelect: false,
+            titleSeperator: ',',
+            options: [
+              {
+                value: 'r50k_base',
+                displayName: 'r50k_base (gpt-3)',
+              },
+              {
+                value: 'p50k_base',
+                displayName: 'p50k_base (gpt-3)',
+              },
+              {
+                value: 'p50k_edit',
+                displayName: 'p50k_edit (gpt-3)',
+              },
+              {
+                value: 'cl100k_base',
+                displayName: 'cl100k_base (gpt-4, gpt-3.5-turbo, gpt-35-turbo)',
+              },
+              {
+                value: 'cl200k_base',
+                displayName: 'cl200k_base (gpt-4o)',
+              },
+            ],
+          },
+        },
+        tokenSize: {
+          title: 'Token size',
+          type: 'integer',
+          description: 'The maximum number of tokens per chunk. 100 tokens equals about 75 words.',
+          minimum: 1,
+          default: 5000,
+          maximum: 8000,
+        },
+        pageOverlapLength: {
+          title: 'Page overlap length',
+          type: 'integer',
+          description: 'The number of chracters to overlap between pages.',
+          minumum: 0,
+          default: 0,
+        },
+      },
+      required: ['chunkingStrategy', 'text', 'encodingModel', 'tokenSize', 'pageOverlapLength'],
+    },
+
+    isInputsOptional: false,
+    isOutputOptional: false,
+    includeRootOutputs: false,
+
+    outputs: {
+      type: 'object',
+      properties: {
+        body: {
+          type: 'object',
+          title: 'Chunked result',
+          description: 'An object containing chunked text.',
+          properties: {
+            value: {
+              type: 'array',
+              title: 'Text items',
+              description: 'Array of chunked text.',
```

Comment:
Does the array have any particular schema or is it just type any?

Reply:
its just an array of strings

Reply:
updated